tests/integration/components/signup-steps/step-one-test.js

```diff
@@ -180,17 +117,16 @@ module('Integration | Component | signup-steps/step-one', function (hooks) {
     assert.dom('[data-test-button=signup]').hasText('Signup');
   });
 
-  skip('role based button should be enabled when all required fields are filled', async function (assert) {
+  test('signup button should be enabled when all required fields are filled', async function (assert) {
```

Comment:
Is there a check to see if the required fields are not filled then it behaves as expected

Reply:
yes here it is 
![image](https://github.com/Real-Dev-Squad/website-www/assets/111434418/087b149f-a89b-4c44-bfe1-803fb2a2f437)